MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,141 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
+
+
+{% tabs %}
+{% highlight C# %}
+foreach (FormField formField in PdfViewer.FormFields) 
+{
+    // Check if the current form field is a text form field 
+    if (formField is TextFormField textBoxField)
+    {
+        // Loop through all the widgets associated with the text box field
+        foreach (var widget in textBoxField.Widgets)
+        {
+            // Set the background color of each widget to yellow
+            widget.BackgroundColor = Colors.Yellow;
+        }
+    }
+}
+ 
+{% endhighlight %}
+{% endtabs %}
+
+### Customize the text color of the form fields
+
+The `ForegroundColor` property sets the color of the text within a form field. This allows you to customize the text color, Text color can be used to highlight important fields or messages in forms. For example, you can set a field's text color to red to draw attention to a mandatory field or a validation error. The following code explains how to enable the ForegroundColor for text form fields. Similarly, you can do the same for other form fields like checkboxes, radio buttons, and other form fields by modifying their respective properties.The following example illustrates how to add a foreground color for a text form field by accessing the text form field’s `ForegroundColor` property. 
+
+
+{% tabs %}
+{% highlight C# %} 
+foreach (FormField formField in PdfViewer.FormFields)
+{
+    // Check if the current form field is a text form field 
+    if (formField is TextFormField textBoxField)
+    {
+        // Loop through all the widgets associated with the text box field
+        foreach (var widget in textBoxField.Widgets)
+        {
+            // Set the foreground color of each widget to Red
+            widget. ForegroundColor = Colors.Red;
+        }
+    }
+} 
+{% endhighlight %}
+{% endtabs %}
+
+
+
+### Customize the border color of the form fields
+
+The `BorderColor` property sets the color of the border around  the form field. For users with color blindness, using high-contrast border colors can improve the visibility of the form fields. Additionally, changing the border color to red for an error state can be more noticeable for those with difficulty distinguishing certain colors. The following code explains how to enable the border color for text form fields. Similarly, you can do the same for other form fields like checkboxes, radio buttons, and other form fields by modifying their respective properties.The following example illustrates how to add a  border color for a text form field by accessing the text form field’s `BorderColor` property. 
+ 
+
+{% tabs %}
+{% highlight C# %} 
+foreach (FormField formField in PdfViewer.FormFields)
+{
```

Comment:
Still the same contradiction occurs
<img width="487" alt="image" src="https://github.com/user-attachments/assets/e9b080a3-d5f1-41de-9d2b-dc34be46efac" />